pages/about.jsx

```diff
@@ -10,17 +10,15 @@ var config = require('../lib/config');
 var Intro = React.createClass({
   render: function () {
     return (
-        <div>
-          <h1>About Mozilla Learning Networks</h1>
-          <section className="intro">
-            <Illustration width={226} height={226}
-            className="img-circle"
-            src1x="/img/pages/about/hive-nyc-summer-quest.jpg"
-            alt="HiveLearningNYC 2014 Hive NYC Summer Quest Maker Party">
-              <p>We want more people to see themselves as citizens of the web. Mozilla Learning Networks offers programs and a global community dedicated to helping people learn the most important skills of our age: <em>the ability to read, write and participate in the digital world.</em></p>
-            </Illustration>
-          </section>
-        </div>
+      <section className="intro">
+        <h1>About Mozilla Learning Networks</h1>
```

Comment:
Oof, changes like this made this PR harder to understand, with all the other changes going on in it :sob: 

If it's still possible, do you think you can actually undo these kinds of changes that are purely indentation changes due to being wrapped in a new element? We could then redo the indentation fixes in a separate PR after the homepage lands... if it's too much trouble though, don't worry about it.